.env.jenkins

```diff
@@ -6,7 +6,7 @@ ETHEREUM_DEV_CLUSTER=1
 EXTENSIONS=1
 FLEET=eth.beta
 GROUP_CHATS_ENABLED=1
-HARDWALLET_ENABLED=0
+HARDWALLET_ENABLED=1
```

Comment:
is the keycard enabled for eveyone if we merge this?

Reply:
@gravityblast it's set for this PR only to allow QA to test this. I will revert that back after testing.